hls4ml/model/optimizer/__init__.py

```diff
@@ -17,10 +18,17 @@
     from hls4ml.model.optimizer.passes.qkeras import OutputRoundingSaturationMode
     from hls4ml.model.optimizer.passes.qkeras import QKerasFactorizeAlpha
     from hls4ml.model.optimizer.passes.qkeras import FuseConsecutiveBatchNormalization
+    from hls4ml.model.optimizer.passes.qkeras import ExtractTernaryThreshold
     __qkeras_optimizers__ = True
 except ImportError:
     __qkeras_optimizers__ = False
 
+if __qkeras_optimizers__:
+    register_pass('output_rounding_saturation_mode', OutputRoundingSaturationMode)
+    register_pass('qkeras_factorize_alpha', QKerasFactorizeAlpha)
+    register_pass('extract_ternary_threshold', ExtractTernaryThreshold)
+    register_pass('fuse_consecutive_batch_normalization', FuseConsecutiveBatchNormalization) 
+
```

Comment:
Changed the order of these so that `QKerasFactorizeAlpha` goes before `ExtractTernaryThreshold` - that might insert `BatchNormalization` layers - go before `FuseConsectuiveBatchNormalization`, which goes before `MergeBatchNormAndQuantizedTanh`.